Packs/PaloAltoNetworks_IoT/Playbooks/playbook-iot-with-servicenow-playbook.yml

```diff
@@ -0,0 +1,357 @@
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "6"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 515
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "4":
+    id: "4"
+    taskid: 2185d265-79b6-4c3c-8072-e07af5df536b
+    type: regular
+    task:
+      id: 2185d265-79b6-4c3c-8072-e07af5df536b
+      version: -1
+      name: Calculate RACI
+      description: IoT RACI model script
+      scriptName: iot-security-get-raci
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "7"
+    scriptarguments:
+      alert_name:
+        simple: ${incident.name}
+      category:
+        simple: ${PaloAltoNetworksIoT.device.category}
+      model:
+        simple: ${PaloAltoNetworksIoT.device.model}
+      profile:
+        simple: ${PaloAltoNetworksIoT.device.profile}
+      raw_type:
+        simple: ${incident.rawType}
+      vendor:
+        simple: ${PaloAltoNetworksIoT.device.vendor}
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 162.5,
+          "y": 835
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "5":
+    id: "5"
+    taskid: 9b974236-9370-4e30-81a0-a3b860afe2a4
+    type: title
+    task:
+      id: 9b974236-9370-4e30-81a0-a3b860afe2a4
+      version: -1
+      name: Done
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 1535
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "6":
+    id: "6"
+    taskid: c8b4ea7e-efc0-410d-820b-e54f4b7e2bc3
+    type: condition
+    task:
+      id: c8b4ea7e-efc0-410d-820b-e54f4b7e2bc3
+      version: -1
+      name: IOT_CONFIG list configured?
+      type: condition
+      iscommand: false
+      brand: ""
+      description: 'to check whether the IOT_CONFIG is defined'
+    nexttasks:
+      '#default#':
+      - "9"
+      "yes":
+      - "4"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isExists
+          left:
+            value:
+              simple: lists.IOT_CONFIG
+            iscontext: true
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 660
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "7":
+    id: "7"
+    taskid: be71d420-9199-41ce-8cee-31f7db39a642
+    type: condition
+    task:
+      id: be71d420-9199-41ce-8cee-31f7db39a642
+      version: -1
+      name: ServiceNow Table Existed?
```

Comment:
How is the table created in context to begin with?

Reply:
The ${PaloAltoNetworksIoT.raci.r_snow.table} is created after the "iot-security-get-raci" step is returning a non-blank r_snow with a table field in it.